allennlp/data/vocabulary.py

```diff
@@ -490,7 +490,7 @@ def _extend(self,
                 pretrained_list = None
             token_counts = list(counter[namespace].items())
             token_counts.sort(key=lambda x: x[1], reverse=True)
-            max_vocab = max_vocab_size[namespace]
+            max_vocab = max_vocab_size.get(namespace)
```

Comment:
Can you add a small test that fails before this change, so we can be sure we don't have a regression sometime on this bug?